README.md

```diff
@@ -45,59 +35,27 @@ Alternatively, if you have the [`pip`](https://pip.pypa.io/) package installed,
 pip3 install axiom-py
```

Comment:
Do we need the other instructions above?

Reply:
Typically I do prefer to include them in python projects as pip can be really weird, but we can remove it if you'd prefer

Reply:
Nah it's fine to keep it if it's useful